S/S2Geography/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+using BinaryBuilder, Pkg
+
+name = "S2Geography"
+version = v"0.1.2"
+sources = [
+    GitSource("https://github.com/paleolimbot/s2geography.git", "26b65bb0a60361adfcc72b0ac427302cbf10b040"),
+    DirectorySource("./bundled"),
+]
+
+# TODO: fix the build
+# by pointing to the right s2 and abseil paths
+# from the JLLs.
+script = raw"""
+cd ${WORKSPACE}/srcdir/s2geography
+atomic_patch -p1 ../patches/msvc_to_win32_target.patch
```

Comment:
This failed because your patch was in `bundled/msvc_to_win32_target.patch` not in `bundled/patches/msvc_to_win32_target.patch` -- I've moved it now